mistralrs-core/src/lib.rs

```diff
@@ -78,6 +78,7 @@ pub struct MistralRsBuilder {
     no_prefix_cache: Option<bool>,
     prefix_cache_n: Option<usize>,
     disable_eos_stop: Option<bool>,
+    gemm_full_precision_f16: Option<bool>,
```

Comment:
I think this would be easier to understand if we make the type `bool`. Is there a benefit to using `Option<bool>`.

Reply:
No reason. I used the same pattern the struct used. Should the other settings change too?